lib/cubit/task_state.dart

```diff
@@ -2,9 +2,9 @@ part of 'task_cubit.dart';
 
 @immutable
 class TaskState extends Equatable {
-  final List<String> taskNames;
-  final String? taskName;
-  final List<String> completedTasks;
+  final List<Task> taskNames;
+  final Task? taskName;
```

Comment:
I am surprised this was switched to a `Task` from a `String`, but if it works this way, cool. Please resolve this comment.

Reply:
Well, I suppose if we build on these we would want to have more fields added (due date, priority, etc) so I thought biting the bullet now would be good.